src/components/AboutPage.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import {Link} from 'react-router';
+import { Link } from 'react-router-dom';
```

Comment:
which formatting does our linter support? Spacing between {} in imports or none? I can't remember. cc: @kwelch.

Reply:
Not sure, I know I prefer with spaces. I also believe prettier default is with spaces.